plugins/Marketplace/Marketplace.php

```diff
@@ -227,6 +229,32 @@ public function removePluginTrialRequest(string $pluginName): void
         }
     }
 
+    public function isInstallAllPaidPluginsVisible(): bool
+    {
+        $consumer = StaticContainer::get('Piwik\Plugins\Marketplace\Consumer');
+        $plugins = StaticContainer::get('Piwik\Plugins\Marketplace\Plugins');
+        $controller = StaticContainer::get('Piwik\Plugins\Marketplace\Controller');
+        $paidPlugins = $plugins->getAllPaidPlugins();
+        $paidPluginsToInstallAtOnce = $controller->getPaidPluginsToInstallAtOnceData($paidPlugins);
+
+        return (
+            $consumer->isValidConsumer() &&
+            Piwik::hasUserSuperUserAccess() &&
+            SettingsPiwik::isAutoUpdatePossible() &&
+            CorePluginsAdmin::isPluginsAdminEnabled() &&
+            count($paidPluginsToInstallAtOnce) > 0
+        );
+    }
+
+    public function getPaidPluginsToInstallAtOnce(): array
+    {
+        $plugins = StaticContainer::get('Piwik\Plugins\Marketplace\Plugins');
```

Comment:
To test also add this here
```suggestion
        return ['ActivityLog'];
        $plugins = StaticContainer::get('Piwik\Plugins\Marketplace\Plugins');
```